packages/gatsby-source-medium/src/__tests__/gatsby-node.js

```diff
@@ -0,0 +1,44 @@
+const axios = require(`axios`)
+const { sourceNodes } = require(`../gatsby-node`)
+
+const fixture = require(`./__fixtures__/medium.json`)
+const fakeMediumData = `])}while(1);</x>${JSON.stringify(fixture)}`
```

Comment:
uuu wat? :P

Is this how actual responses from medium look like? :)

Reply:
Yep Medium is prefixing their API with `])}while(1);</x>` and I don't know why x)